src/gc/bits.d

```diff
@@ -38,24 +39,30 @@ struct GCBits
     wordtype* data;
     size_t nbits;
 
-    void Dtor() nothrow
+    void Dtor(bool mmap) nothrow @nogc
```

Comment:
Could use the same default (mmap = false) as the alloc function.

Reply:
Should it be implicit or explicit?
I worry that someone could forget to explicit the parameter in new code

Reply:
I think is a valid concern, but in any case it should be consistent (all with default or all without it).